extensions/interactions/guppy.html

```diff
@@ -1,24 +1,42 @@
-<link rel="stylesheet" href="/third_party/static/guppy-b5055b/build/guppy-default.min.css">
+<link rel="stylesheet" href="/third_party/static/guppy-f944022/build/guppy-default.min.css">
+<link rel="stylesheet" href="/third_party/static/guppy-f944022/build/guppy-default-osk.min.css">
 <script>
   // Use the normal default browser eventlistener without passive set to false.
   // TODO(#8833): Use passive listeners when guppy supports it.
   EventTarget.prototype.addEventListener = (
     EventTarget.prototype.addEventListener._original);
 </script>
-<script type="text/javascript" src="/third_party/static/guppy-b5055b/build/guppy.min.js"></script>
+<script type="text/javascript" src="/third_party/static/guppy-f944022/build/guppy.min.js"></script>
+<script type="text/javascript" src="/third_party/static/guppy-f944022/build/guppy_osk.min.js"></script>
 
 <script type="text/javascript">
+  Guppy.use_osk(new GuppyOSK({"goto_tab":"arithmetic"}));
```

Comment:
Use single quotes instead of double quotes in this script tag.

Reply:
Done.